@uportal/notification-list/src/components/NotificationList.vue

```diff
@@ -116,7 +117,7 @@ export default {
         this.fetchNotificationData();
 
         const queryParams = queryString.parse(location.search);
-        this.highlight = queryParams.highlight;
+        this.highlight = queryParams.highlight || queryParams.pP_highlight;
```

Comment:
`pP_highlight` ? where is this value coming from?

Reply:
uPortal converts `highlight` to pP_highlight when it catches the path to the notification list. I was intending to ask Drew about this to see if there is a better way of handling it.

Reply:
/cc @drewwills

Reply:
I've discussed this with @drewwills and he explained that is is a necessary evil atm due to uPortal's inner workings around query parameters.